### PR TITLE
okd-machine-os: update base images

### PR DESCRIPTION
**Pull request by vrutkovs**
Use 4.13/4.14 artifacts

### CODE REVIEW

File: ci-operator/config/openshift/okd-machine-os/openshift-okd-machine-os-master.yaml [hunk 1]
```diff
@@ -19,6 +19,9 @@ images:
   to: okd-rpms
 - dockerfile_path: Dockerfile.generated
   inputs:
+    artifacts:
```
Comment (LorbusChris) [inline ci-operator/config/openshift/okd-machine-os/openshift-okd-machine-os-master.yaml:22]: I guess the `origin_4.14_artifacts` base image has to be renamed to `artifacts` here too? Same in the v4.14 config

Comment (vrutkovs) [reply ci-operator/config/openshift/okd-machine-os/openshift-okd-machine-os-master.yaml:22]: Good idea, done

**Comment by vrutkovs** [comment]
/pj-rehearse

lets try it again

**Comment by LorbusChris** [comment]
/shrug
Changes look sane, let's get this in
/rehearsals-ack

**Pull request merged**